src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -143,6 +144,11 @@ private void fillInnerParts() {
     private void setWindowDefaultSize(GuiSettings guiSettings) {
         primaryStage.setHeight(guiSettings.getWindowHeight());
         primaryStage.setWidth(guiSettings.getWindowWidth());
+
+        if (guiSettings.getIsMaximized()) {
+            primaryStage.setMaximized(true);
```

Comment:
Would it be better to replace this with primaryStage.setMaximized(guiSettings.getIsMaximized())?

Reply:
Yes, it has been changed